rustls/src/crypto/mod.rs

```diff
@@ -74,6 +75,25 @@ pub use crate::msgs::handshake::KeyExchangeAlgorithm;
 /// This structure provides defaults. Everything in it can be overridden at
 /// runtime by replacing field values as needed.
 ///
+/// # Using the per-process default `CryptoProvider`
+///
+/// There is the concept of an implicit default provider, configured at run-time once in
+/// a given process.
+///
+/// It is used for functions like [`ClientConfig::builder()`] and [`ServerConfig::builder()`].
+///
+/// The intention is that an application can specify the [`CryptoProvider`] they wish to use
+/// once, and have that apply to the variety of places in their application does TLS
+/// (which may be wrapped inside other libraries).
```

Comment:
another small readability tweak:
```suggestion
/// The intention is that an application can specify the [`CryptoProvider`] they wish to use
/// once, and have that apply to the variety of places where their application does TLS
/// (which may be wrapped inside other libraries).
```